crates/matrix-sdk/src/encryption/backups/mod.rs

```diff
@@ -423,6 +423,39 @@ impl Backups {
         Ok(())
     }
 
+    /// Deletes the current backup version from the server
```

Comment:
I think that this need a bit more of a warning, feel free to add some ⚠️ symbols. We should probably also link to the safer, `disable()` mehod from, and to, this one.

As usual, an doc example wouldn't hurt either.